examples/platform_view/lib/main.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017, the Flutter project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() {
+  runApp(new PlatformView());
+}
+
+class PlatformView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
+      title: 'Platform View ',
+      theme: new ThemeData(
+        primarySwatch: Colors.grey,
+      ),
+      home: new MyHomePage(title: 'Platform View'),
+    );
+  }
+}
+
+class MyHomePage extends StatefulWidget {
+  MyHomePage({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  _MyHomePageState createState() => new _MyHomePageState();
+}
+
+class _MyHomePageState extends State<MyHomePage> {
+  static const MethodChannel _methodChannel =
+      const MethodChannel("samples.flutter.io/platform_view");
+
+  int _counter = 0;
+
+  void _incrementCounter() {
+    setState(() {
+      _counter++;
+    });
+  }
+
+  Future<Null> _launchPlatformCount() async {
+    final int platformCounter =
+        await _methodChannel.invokeMethod("switchView", _counter);
+    setState(() {
+      _counter = platformCounter;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) => new Scaffold(
```

Comment:
Indentation in this method could be improved. Hopefully enough to insert some extra `,`s and then run `dartfmt`.